src/dht/find-peer.js

```diff
@@ -27,14 +27,15 @@ module.exports = (common, options) => {
     after(() => common.clean())
 
     it('should find other peers', async () => {
-      const res = await nodeA.dht.findPeer(nodeB.peerId.id)
-
+      const nodeBId = await nodeB.id()
```

Comment:
Why not use `nodeB.peerId` here?

Reply:
It's not part of the API defined in `interface-js-ipfs-core` so we should really remove it from `ipfsd-ctl` return values.

This is my mind wandering from the task in hand though so I'm happy to change it if required.

Reply:
Right, so it shouldn't be used in the interface tests because we can't guarantee `ipfsd-ctl` spawned the node(s) in the test. 👍 Sounds like a good plan, I'm happy to leave this like it is. Do you want to open an issue here to track that task?

Reply:
I've opened https://github.com/ipfs/interface-js-ipfs-core/issues/582